packages/web-app-files/src/components/Collaborators/Collaborator.vue

```diff
@@ -61,9 +61,9 @@
             <div v-if="$_reshareInformation">
               <oc-tag
                 :id="$_resharerToggleId"
+                v-oc-tooltip="$gettext('Show resharer details')"
```

Comment:
Does this tag need a tooltip at all? Usability wise?

Reply:
Wouldn't say so. There's tags that are links and there I find tooltips helpful, if it's "just" a small hint I would omit them